recipes/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "torsiondrive" %}
```

Comment:
This file needs to be in `recipes/torsiondrive/meta.yaml`, not directly in `recipes`.

Reply:
👌